src/include/compat.h

```diff
@@ -14,8 +14,13 @@
 
 #include "acconfig.h"
 
+#define PROCPREFIX ""
```

Comment:
@tchaikov 
I first considered doing that.

But I liked the definition under one FreeBSD clause better.
And it allows for other OSes to do the same.

But if you feel strong about this, I don't mind changing